version.properties

```diff
@@ -1,12 +1,12 @@
-#Thu, 15 Jul 2021 21:23:44 +0200
+#Mon, 19 Jul 2021 08:38:28 -0400
 
 # WordPress version information
-wordpress.versionName=17.8-rc-2
-wordpress.versionCode=1077
+wordpress.versionName=17.8-rc-3
+wordpress.versionCode=1079
 
-wordpress.zalpha.versionName=alpha-306
-wordpress.zalpha.versionCode=1078
+wordpress.zalpha.versionName=alpha-307
+wordpress.zalpha.versionCode=1080
 
 # Jetpack version information
-jetpack.versionName=17.8-rc-2
-jetpack.versionCode=1076
+jetpack.versionName=17.8-rc-3
+jetpack.versionCode=1077
```

Comment:
I think you meant it's used in `WordPress 17.8-rc-3` since the end result of this ^ change is `17.8-rc-3`. With that assumption in mind, I believe the answer is yes, because it's a different app. However, worth pinging @AliSoftware for when he gets back, just so he can double check the intent.

@AliSoftware I don't know why these versions were separated (although I reviewed PRs about it, I can't remember the reason), so it's worth thinking about using a single version & version code for Jetpack and WordPress apps if we are going to always keep them in sync.

Reply:
Yep:
- @mokagio it's different apps in PlayStore so it's ok to use the same versionCode indeed.
- @oguzkocer That's a leftover of Javon's changes where he split the versioning for WP and JP, which I intend to revert. Now we use a single version and version code for JP and WP, so yes, definitively, there are plans to unify them and using a single entry there for it instead of having to keep them separate but update them both to the same value.

I have yet to find some time to pick that project back up (needed to focus on my other hotfix lanes project before AFK, and I'm hoping to tackle that revert of Javon's changes in release toolkit in this Sprint or the next), hopefully once this is tackled things will be way cleaner there.